### DISCUSSION TITLE
Legacy versions of nHibernate v1.x older than v1.2

### DISCUSSION THREAD

**Discussion by mas1701** [nhibernate/nhibernate-core#3627] [Q&A] [answered]
For editing a legacy software project from many years ago, I'm looking for nHibernate v1.0 or v1.1 (not sure which one is actually needed), it's for a co-worker of mine. I have only found v1.2 on GitHub. Any additional sources where I could find it?

**Comment by oskarb** [comment, marked as answer]
If you mean the source, the tags seem to exist in git.

If you mean binaries, this pre-dates nuget. Non-nuget binaries were (and still are) published on Sourceforge:
https://sourceforge.net/projects/nhibernate/files/

It seems you want the "OldFiles" folder for such palaeolithical artifacts.

**Reply by mas1701** [reply]
Great, thanks.
